packages/wagmi/package.json

```diff
@@ -45,12 +45,12 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "typescript": "^4.9.5",
-    "wagmi": "0.12.7"
+    "wagmi": "0.12.7 || <=0.12.13"
```

Comment:
could we consistency to keep versions in `devDependency` in a format without using the `or`?
```
"wagmi": "0.12.7 || <=0.12.13"
```